testdata/go/v2/memcached-operator/config/scorecard-testbundle/kustomization.yaml

```diff
@@ -0,0 +1,13 @@
+resources:
+- ../scorecard
+patchesJson6902:
+# This patch is commented so you can make modifications to your kuttl tests
+# before enabling them to run with scorecard. If you make any changes to a CRD,
+# make sure those changes are reflected in kuttl test cases before uncommenting this patch.
+#- path: patches/kuttl.config.yaml
```

Comment:
What does this comment mean? If I create a new api then, should I need to update it manually? Cannot we use this plugin in the create api to update this file automatically?